docs/source/concepts/population.rst

```diff
@@ -9,8 +9,77 @@ Population Management
    :local:
    :backlinks: none
 
-Outline
--------
- - Initialization of simulants and the dependency problem.
- - Managing access with population views.
- -
+Since ``Vivarium`` is an agent-based simulation framework, managing a group of
+:term:`simulants` and their attributes is a critical task. Fundamentally, to run
+a simulation we need to be able to create new simulates, update their state
+attributes, and facilitate access to their state so that :term:`components` in
+the simulation can do interesting things based on it. The tooling to support
+working with our simulant population is called the population management system.
+
+The State Table
+---------------
+
+The core representation of simulants and their state information in ``Vivarium``
+is a :class:`pandas.DataFrame <Data Frame>` known as the state table. Under this
+representation rows represent simulants while columns correspond to state
+attributes like age, sex or systolic blood pressure. These columns represent one
+of several important resources within ``Vivarium`` that other components can
+draw on. Each of the actions we need to be able to take correspond to a
+manipulation of this state table. The addition of new simulants is the creation
+of rows, the creation of new state attributes is the creation of columns, and
+the reading and updating of state is reading and updating the dataframe itself.
+
+<<TODO: image of state table w/ expanding rows and columns>>
+
+Population Views
+----------------
```

Comment:
Instead of a section about views, have a section about accessing and updating the population.  Start out with why it's important framed in the context of time steps.  A picture of two state tables would be good.  E.g. I need to read these columns of X_t to update these columns of X_t+1. Then talk about the implementation (and how it relates to the idea that components map to sets of attributes/columns).  Code examples of how to get one.  How to get subviews.  How to query.  I think relating them to accessing SQL tables might be helpful (SELECT and WHERE map reasonably well to subview and get).  Talk about the Markov property (we shouldn't be using the state table to communicate between components during the same time step and warn that this will be enforced).  Something brief and reference to a new concept stub.

Again separate PR.